README.md

```diff
@@ -0,0 +1,45 @@
+# av_gps
+
+This repository hosts the required environment for running the ROS 2 Novatel GPS driver, specifically configured for running with a Novatel PwrPak7D GNSS receiver with dual antennas.
+
+
+## Docker container
+- **Operating System**: Ubuntu 22.04
+- **ROS Distribution**: Humble Hawksbill
+- **Velodyne Drivers**:
+    - `ros-humble-novatel-gps-driver ` ( version `4.1.2-1`): Manages communication with the Novatel GPS receiver.
```

Comment:
Given we are not actually pinning apt deps versions in the Dockerfile, should we bother tracking this version here?

Reply:
Done :)